### ISSUE TITLE
Sled Recipes not working for Fabric 1.20.4

### ISSUE THREAD

**Issue by BioMasterZap** [MehVahdJukaar/Christmas#86]
With v2.3.4 for Fabric 1.20.4, the Sled recipes do not seem to be working properly. With REI, all the sleds showed the same recipe using Oak Planks. Without any other mods in my test profile, the only planks that can craft a Sled are Oak and it makes a Spruce Sled. It should be easy enough for me to fix with a data pack, but I didn't see any other reports for this issue.

**Comment by MCRusher** [comment]
yup. I also have wilder wild installed and so every sled wood variant recipe is made with oak planks but it makes a palm sled for some reason
